lib/DDG/Spice/TranslateOneWord.pm

```diff
@@ -0,0 +1,35 @@
+package DDG::Spice::TranslateOneWord;
```

Comment:
Whoops, this shouldn't be in here. 

I think you'll want to move this code to a separate Git branch.